lib/ansible/modules/notification/mail.py

```diff
@@ -264,7 +265,10 @@ def main():
     try:
         if secure != 'never':
             try:
-                smtp = smtplib.SMTP_SSL(host=host, port=port, timeout=timeout)
+                if PY3:
```

Comment:
I like the approach from this PR https://github.com/ansible/ansible/pull/49016/files#diff-51726da56deae6bf8b52a885b2b0d2afR268 better style-wise. It reduces duplication of the function call.

Reply:
I disagree, I find it less readable for no real gain.

Reply:
dagwieers +1  locality makes this version more readable with very little sacrifice of consiseness.

The answer would be different if there were many conditionals or optional arguments to add but neither of those is the case here.

Reply:
Agreed.